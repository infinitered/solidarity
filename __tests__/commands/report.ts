jest.mock('envinfo')

import reportCommand from '../../src/commands/report'
import { print as envinfoPrint } from 'envinfo'

test('Snapshot check help command', () => {
  expect(reportCommand).toMatchSnapshot()
})

it('enforces required properties', () => {
  expect(reportCommand.description).toBeTruthy()
  expect(reportCommand.run).toBeTruthy()
  expect(typeof reportCommand.run).toBe('function')
})

// test('check solidarity report', async () => {
//   const result = await reportCommand.run()
//   expect(envinfoPrint.mock.calls.length).toBe(1)
// })
