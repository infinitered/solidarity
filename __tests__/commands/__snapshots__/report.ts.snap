// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot check help command 1`] = `
Object {
  "alias": "r",
  "description": "Report an issue",
  "run": [Function],
}
`;
